FSharpTests.Directory.Build.props

```diff
@@ -33,8 +33,8 @@
   <!-- SDK targets override -->
   <PropertyGroup>
     <_FSharpBuildTargetFramework Condition="'$(FSharpTestCompilerVersion)' == 'net40'">net46</_FSharpBuildTargetFramework>
-    <_FSharpBuildTargetFramework Condition="'$(FSharpTestCompilerVersion)' == 'coreclr'">netstandard2.0</_FSharpBuildTargetFramework>
-    <_FSharpBuildBinPath>$(MSBuildThisFileDirectory)artifacts\bin\FSharp.Build\$(Configuration)\$(_FSharpBuildTargetFramework)</_FSharpBuildBinPath>
+    <_FSharpBuildTargetFramework Condition="'$(FSharpTestCompilerVersion)' == 'coreclr'">netcoreapp2.1</_FSharpBuildTargetFramework>
+    <_FSharpBuildBinPath>$(MSBuildThisFileDirectory)artifacts\bin\fcs\$(Configuration)\$(_FSharpBuildTargetFramework)</_FSharpBuildBinPath>
```

Comment:
I needed this change to load the FSharp.Build.dll from one right place.  

I think it should always load from the `fcs` directory (or better the `Bootstrap` directory?)